core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -123,6 +123,7 @@
     public static final String PATIENT_VIEW_DIGITAL_SLIDE_META_URL = "digitalslidearchive.meta.url";
     public static final String PATIENT_VIEW_TCGA_PATH_REPORT_URL = "tcga_path_report.url";
     public static final String ONCOKB_URL = "oncokb.url";
```

Comment:
maybe also rename it to oncokb.api.url so that we can reset it for the instances that have set a oncokb.url already.